Automattic/woocommerce-payments
# WooPay naming/styling updates

Fixes 635-gh-Automattic/woopay

#### Changes proposed in this Pull Request

* Update registration update copy (see 635-gh-Automattic/woopay)
* Replace Platform Checkout with WooPay in the UI
* Replace the placeholder Woo logo on the settings page to use the WooPay logo
* Use production URLs as the default in `PLATFORM_CHECKOUT_FRONTEND_HOST` and `PLATFORM_CHECKOUT_HOST`

Context: p1649353944790879-slack-C022WMN88KG, pdpOw2-yo-p2








#### Testing instructions





* Make sure platform checkout is enabled for your merchant site.
* Ensure that WooPay registration UI matches the updated design in pdpOw2-yo-p2. Note that the link in "powered by WooPay" is not set on this PR, this will be set later.
* When `PLATFORM_CHECKOUT_FRONTEND_HOST` and `PLATFORM_CHECKOUT_HOST` is not set, WCPay attempts to load the OTP iframe from `woo.app`
* WooCommerce -> Settings -> Payments -> Express Checkout page uses the correct logo cO37XZJMyjQweZ6RhEeDhX-fi-31%3A25535

-------------------


**Post merge**
